src/core/test/system/system.t.sol

```diff
@@ -16,9 +16,24 @@
 pragma solidity >=0.5.12;
 
 import "./system.sol";
+import "./users/borrower.sol";
 
 contract STest is SystemTest {
```

Comment:
What is STest just checking if systemTest works?

Reply:
STest are our old system tests, we will get rid of this file eventually